services/resource-usage-tracker/tests/unit/with_dbs/conftest.py

```diff
@@ -62,3 +70,75 @@ def mocked_prometheus(mocker: MockerFixture) -> mock.Mock:
         autospec=True,
     )
     return mocked_get_prometheus_api_client
+
+
+@pytest.fixture()
+def user_id() -> UserID:
+    return UserID(randint(1, 10000))
```

Comment:
MINOR TIP: IMO using `faker: Fake` makes all fake values more uniform and at the same time you learn about new ways to fake data https://faker.readthedocs.io/en/latest/

Reply:
i will refactor/add additional tests in upcoming PR